### PR TITLE
Huntress, Look-Alike and the Priest!

### PR DESCRIPTION
**Pull request by ryandezinator**

### CODE REVIEW

File: interpretation/ww_head.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
     # =============================================================
     if isPeasant == True:
         if is_command(message,['warn'],False,unip):
-            # Warn the user that they've been active for a while.
+            # Warn the user that they've been inactive for a while.
```
Comment (romangraef) [inline interpretation/ww_head.py]: Belongs to a different commit, but ok

Comment (ryandezinator) [reply interpretation/ww_head.py]: I just saw that and thought it was weird, so changed it
